pkgs/applications/misc/openbangla-keyboard/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, cmake
+, pkg-config
+, cargo
+, rustc
+, rustPlatform
+, ibus
+, qtbase
+, zstd
+}:
+
+mkDerivation rec {
+  pname = "openbangla-keyboard";
+  version = "unstable-2022-04-27";
+
+  src = fetchFromGitHub {
+    owner = "openbangla";
+    repo = "openbangla-keyboard";
+    rev = "adab38f09d47b8b6c88f57256e89983152b8384c";
+    hash = "sha256-z63bsrxhUudb8O29genHmcOy4pzCO+KzadUc7Hs0ULY=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    cargo
+    rustc
+    rustPlatform.cargoSetupHook
```

Comment:
```suggestion
    rustPlatform.cargo
    rustPlatform.rustc
    rustPlatform.cargoSetupHook
```

Reply:
I don't think `rustPlatform` has these two attributes, [`makeRustPlatform`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/rust/make-rust-platform.nix) does not list them and tab complete in the repl doesnt either

```
nix-repl> a = import ./. { } 

nix-repl> a.rustPlatform.
a.rustPlatform.bindgenHook        a.rustPlatform.cargoCheckHook     a.rustPlatform.fetchCargoTarball  a.rustPlatform.rust
a.rustPlatform.buildRustPackage   a.rustPlatform.cargoInstallHook   a.rustPlatform.importCargoLock    a.rustPlatform.rustLibSrc
a.rustPlatform.cargoBuildHook     a.rustPlatform.cargoSetupHook     a.rustPlatform.maturinBuildHook   a.rustPlatform.rustcSrc

```

Reply:
They are under rustPlatform.rust.

Reply:
thanks